system/data-privacy.rst

```diff
@@ -118,26 +125,36 @@ Frequently Asked Questions
    in the Admin Panel under **System > Data Privacy >
    Completed Tasks > Delete User > Deleted Tickets**.
 
-🤔 What about user information stored in internal notes or other messages?
+What about user information stored in internal notes or other messages?
    The deletion process removes **user accounts and associated tickets only**.
 
-   If there are references to a user’s name or information
+   If there are references to a user's name or information
    stored elsewhere in the system,
    that information **will not be removed**
    because there is no way to safely determine
    if it actually describes the user in question.
 
-😵 I deleted an user and can still see a message they sent!
+I deleted an user and can still see a message they sent!
    Tickets can only belong to a single customer,
    but may contain messages (“articles”) from many people.
-   If you deleted a user but you’re still seeing articles they sent,
-   don’t worry—those articles are for a ticket that belongs to someone else,
-   and no longer contain any reference to the sender’s identity.
+   If you deleted a user but you're still seeing articles they sent,
+   don't worry—those articles are for a ticket that belongs to someone else,
+   and no longer contain any reference to the sender's identity.
 
-🚮 I removed a customer, now my reporting is off!
+I removed a customer, now my reporting is off!
    When removing users and their tickets, all references are removed.
    This also affects e.g. Reporting - these information are lost.
 
-🤔 How long does Zammad store created tasks?
+How long does Zammad store created tasks?
    Please see the on-premise data section of the
    :docs:`data privacy </appendix/privacy.html>` chapter.
+
+What about re-assigned tickets? I want to delete them, too.
+   Only tickets assigned to the matching user at the time of the execution of
```

Comment:
In wonder if we should be more specific that this only applies to customer assignments, not owners 🤔 Perhaps we could just change to `matching user` to `matching customer` instead.

Remember, deletion tasks will reassigned owner tickets to the system user, they will not be deleted!